src/dev/graph/layout.rs

```diff
@@ -203,6 +221,34 @@ impl CircuitLayout {
             draw_cell(&root, column_index(&column), row)?;
         }
 
+        // Mark equality-constrained cells.
+        if self.mark_equality_cells {
+            for (l_col, l_row, r_col, r_row) in &layout.equality {
+                let l_col = column_index(l_col);
+                let r_col = column_index(r_col);
+                root.draw(&Rectangle::new(
+                    [(l_col, *l_row), (l_col + 1, *l_row + 1)],
+                    ShapeStyle::from(&RED).filled(),
+                ))?;
+                root.draw(&Rectangle::new(
+                    [(r_col, *r_row), (r_col + 1, *r_row + 1)],
+                    ShapeStyle::from(&RED).filled(),
+                ))?;
```

Comment:
It would be nice to have a border around each cell, so that it's easier to count them (similar to assigned cells).